### ISSUE TITLE
Bug: Template Won't Launch From Winlink Express

### ISSUE THREAD

**Issue by nojronatron** [nojronatron/Bigfoot-Bib-Report-WL-Form#15]
File Bigfoot Bib Report.txt has the required `Form:` declaration but somehow cannot properly read the file as-is.

The fix is to add line-feeds between each line in the template file to ensure there is no incidental concatenation.

Update the file so each entry has an additional CR/LF character between each Template keyword and following argument(s) for proper operation.

**Comment by nojronatron** [comment]
Working to resolve this issue today.

**Comment by jhmartin** [comment]
Still seeing this in 101b. The error is it can't open Form: formname here <newlinenewline> To: <var

**Comment by nojronatron** [comment]
There is some sort of blank-line issue going on, probably related to one of my dev environments being Linux. Try this:
1. Open Winlink Express
2. Click Message > Template Manager
3. Open (your callsign) Templates
4. Click Bigfoot Bib Report.txt
5. Click Edit

Are there any blank lines? Or is everything smooshed together in a single line?

If smooshed together, add blank lines so the file looks like this:

```
Form: Bigfoot Bib Report Initial.html

To: <var address>

Subject:<var msgsubject>

Msg: 
<var TheData>

-----

<var Comment>

<var FormatNote>

-----

Winlink Express Sender <var MsgSender>

Template version <var Theversion>
```

Let me know what happens and I'll figure out a way to fix this in the next bugfix release.

**Comment by jhmartin** [comment]
I'll try that. Perhaps run unix2dos on the file before comitting it?

**Comment by nojronatron** [comment]
Thanks! Since I have used both linux and windows dev environments, it probably has to do with this [git behavior mentioned in stackoverflow](https://stackoverflow.com/questions/1967370/git-replacing-lf-with-crlf/20653073#20653073). Next bugfix forthcoming.

**Comment by jhmartin** [comment]
The process above worked around the issue.

**Comment by nojronatron** [comment]
It seems like GitHub's Release ZIP process is stripping CR characters.
Follow the instructions in [this post](https://github.com/nojronatron/Bigfoot-Bib-Report-WL-Form/issues/15#issuecomment-1200459504) and the problem should go away.
Meanwhile, I'll try to find a better release system that will retain the CRLF settings so Winlink Express doesn't misinterpret the file on read/import.

**Comment by jhmartin** [comment]
Although not a short-term solution, might inquire with the RMS people if winlink can be made more tolerant on reading the file.

**Comment by DaveWilma** [comment]
My TXT file looks good, plenty of line feeds. Still no joy.

**Comment by jhmartin** [comment]
@DaveWilma Did you look in the Template Manager, or in notepad? Notepad etc don't show the issue.

**Comment by nojronatron** [comment]
> 

If you are using Template Manager to add the line feeds, then please attach a screen shot of the template manager so we can debug it by eyeball.

**Comment by DaveWilma** [comment]
That did it, Edit from the Template manager. I am looking at the template.

**Comment by nojronatron** [comment]
Fixed with PR #20

**Comment by bytescream** [comment]
Wow, who would have thought the LF/CRLF issue would be the challenge here.

Any way I can help. 

Been following, and sure enough, simple Visual Studio Code fix is to select CRLF and save. 
Sorry it is not that simple in GitHub build.
